test/e2e/image_scp_test.go

```diff
@@ -0,0 +1,147 @@
+package integration
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/containers/common/pkg/config"
+	. "github.com/containers/podman/v3/test/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("podman image scp", func() {
+	ConfPath := struct {
+		Value string
+		IsSet bool
+	}{}
+	var (
+		tempdir    string
+		podmanTest *PodmanTestIntegration
+	)
+
+	BeforeEach(func() {
+		ConfPath.Value, ConfPath.IsSet = os.LookupEnv("CONTAINERS_CONF")
+		conf, err := ioutil.TempFile("", "containersconf")
+		if err != nil {
+			panic(err)
+		}
+		os.Setenv("CONTAINERS_CONF", conf.Name())
+		tempdir, err = CreateTempDirInTempDir()
+		if err != nil {
+			os.Exit(1)
+		}
+		podmanTest = PodmanTestCreate(tempdir)
+		podmanTest.Setup()
+	})
+
+	AfterEach(func() {
+		podmanTest.Cleanup()
+		os.Remove(os.Getenv("CONTAINERS_CONF"))
+		if ConfPath.IsSet {
+			os.Setenv("CONTAINERS_CONF", ConfPath.Value)
+		} else {
+			os.Unsetenv("CONTAINERS_CONF")
+		}
+		f := CurrentGinkgoTestDescription()
+		processTestResult(f)
+
+	})
+
+	It("podman-remote image scp oci flag", func() {
+		if !IsRemote() {
+			Skip("Remote only test")
+		}
+		podmanTest.StopRemoteService()
+		podmanTest.StartRemoteService()
+		cmd := []string{"system", "connection", "add",
+			"--default",
+			"remote",
+			"unix:///run/podman/podman.sock",
+		}
+		session := podmanTest.Podman(cmd)
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).Should(Equal(0))
+
+		scp := podmanTest.Podman([]string{"image", "scp", "--format", "oci-archive", ALPINE})
+		scp.WaitWithDefaultTimeout()
+		Expect(scp.ExitCode()).To(Equal(0))
+	})
+
+	It("podman image scp quiet flag", func() {
+		if IsRemote() {
+			podmanTest.StopRemoteService()
+			podmanTest.StartRemoteService()
+		}
+		cmd := []string{"system", "connection", "add",
+			"--default",
+			"remote",
+			"unix:///run/podman/podman.sock",
+		}
+		session := podmanTest.Podman(cmd)
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).Should(Equal(0))
+
+		scp := podmanTest.Podman([]string{"image", "scp", "-q", ALPINE})
+		scp.WaitWithDefaultTimeout()
+		Expect(scp.ExitCode()).To(Equal(0))
```

Comment:
This pattern is anti-recommended, please see #10997. Could you fix it (and all instances) to use the recommended form (`Expect(scp).To(Exit(0))`)? Thank you!

Reply:
@edsantiago is it `To(Exit(0)` or `Should(Exit(0)`? Or is either okay?

Reply:
According to the [docs I found](https://onsi.github.io/gomega/#making-assertions), it looks like they're equivalent.

Reply:
Ok cool, trying to switch now as well as on another feature I am doing but I am being told that `Exit` is an undeclared name. Is there a rebase I need to do to be able to use this syntax?

Reply:
Oh yeah; you need to also include `gomega/gexec` in addition to `gomega`

Reply:
Thanks, sorry for letting the other format through in my pid PR last week

Reply:
Our PRs were both in flight at the same time. How were you to know? It's not a problem at all!